misc/triage/labels.yaml

```diff
@@ -0,0 +1,142 @@
+---
+repo: aquasecurity/trivy
+labels:
+
+# kind
+- name: kind/api-change
+  color: e11d21
+  description: Categorizes issue or PR as related to adding, removing, or otherwise changing an API
+- name: kind/bug
+  color: e11d21
+  from: bug
+  description: Categorizes issue or PR as related to a bug.
+- name: kind/cleanup
+  color: c7def8
+  description: Categorizes issue or PR as related to cleaning up code, process, or technical debt.
+- name: kind/deprecation
+  color: e11d21
+  from: enhancement
+  description: Categorizes issue or PR as related to a feature/enhancement marked for deprecation.
+- name: kind/documentation
+  color: c7def8
+  description: Categorizes issue or PR as related to documentation.
+- name: kind/feature
+  color: c7def8
+  description: Categorizes issue or PR as related to a new feature.
+- name: kind/testing
+  color: c7def8
+  description: Categorizes issue or PR as related to a unit/integration test.
+- name: kind/regression
+  color: e11d21
+  description: Categorizes issue or PR as related to a regression from a prior release.
+- name: kind/security
+  color: f4dddc
+  description: security issues
+
+# lifecycle
+- name: lifecycle/active
+  color: 8fc951
+  description: Indicates that an issue or PR is actively being worked on by a contributor.
+- name: lifecycle/frozen
+  color: d3e2f0
+  description: Indicates that an issue or PR should not be auto-closed due to staleness.
+- name: lifecycle/rotten
+  color: "604460"
+  description: Denotes an issue or PR that has aged beyond stale and will be auto-closed.
+- name: lifecycle/stale
+  color: "795548"
+  description: Denotes an issue or PR has remained open with no activity and has become stale.
+
+# priority
+- name: priority/awaiting-more-evidence
+  color: fef2c0
+  description: Lowest priority. Possibly useful, but not yet enough support to actually get it done.
+- name: priority/backlog
+  color: fbca04
+  description: Higher priority than priority/awaiting-more-evidence.
+- name: priority/critical-urgent
+  color: e11d21
+  description: Highest priority. Must be actively worked on as someone's top priority right now.
+- name: priority/important-longterm
+  color: eb6420
+  description: Important over the long term, but may not be staffed and/or may need multiple releases to complete.
+- name: priority/important-soon
+  color: eb6420
+  description: Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
```

Comment:
What's the difference between something that's important but not critical? How about the difference between `important-soon` and `critical-urgent`? Which is more important? Shouldn't everything be considered important if its not `awaiting-more-evidence` or in the `backlog`?

Maybe we can simplify to 3 levels: `awaiting-more-evidence` -> `backlog`-> `important`. In this order.

Thoughts?

Reply:
It is mentioned here.
https://github.com/aquasecurity/trivy/pull/473/files#diff-71b35dfc9f3592887caa6f2ec1592dbeR69-R77